src/pal/src/config.h.in

```diff
@@ -101,6 +101,10 @@
 #cmakedefine01 PTHREAD_CREATE_MODIFIES_ERRNO
 #cmakedefine01 SEM_INIT_MODIFIES_ERRNO
 #cmakedefine01 HAVE_PROCFS_CTL
+#cmakedefine01 HAVE_PROCFS_MEM
+#cmakedefine01 HAVE_PROCFS_MAPS
+#cmakedefine01 HAVE_PROCFS_STAT
+#cmakedefine01 HAVE_PROCFS_STATUS
```

Comment:
These are necessary to get ReadProcessMemory to work correctly at least HAVE_PROCFS_MEM, but the rest are all about correctness and I think related enough.

From: noahfalk [mailto:notifications@github.com]
Sent: Sunday, May 22, 2016 5:19 PM
To: dotnet/coreclr <coreclr@noreply.github.com>
Cc: Mike McLaughlin <mikem@microsoft.com>; Assign <assign@noreply.github.com>
Subject: Re: [dotnet/coreclr] Change the dbgshim launch handshake back (#5151)


In src/pal/src/config.h.in<https://github.com/dotnet/coreclr/pull/5151#discussion_r64159360>:

> @@ -101,6 +101,10 @@

>  #cmakedefine01 PTHREAD_CREATE_MODIFIES_ERRNO

>  #cmakedefine01 SEM_INIT_MODIFIES_ERRNO

>  #cmakedefine01 HAVE_PROCFS_CTL

> +#cmakedefine01 HAVE_PROCFS_MEM

> +#cmakedefine01 HAVE_PROCFS_MAPS

> +#cmakedefine01 HAVE_PROCFS_STAT

> +#cmakedefine01 HAVE_PROCFS_STATUS

Did these changes relate to the bug fix?

—
You are receiving this because you were assigned.
Reply to this email directly or view it on GitHub<https://github.com/dotnet/coreclr/pull/5151/files/41ca2d79992390fa2847629f34c30479644a9b7d#r64159360>